app/uk/gov/hmrc/bindingtariffrulingfrontend/audit/AuditService.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.bindingtariffrulingfrontend.audit
+
+import javax.inject.{Inject, Singleton}
+import uk.gov.hmrc.bindingtariffrulingfrontend.model.Ruling
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.bootstrap.audit.DefaultAuditConnector
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+@Singleton
+class AuditService @Inject()(auditConnector: DefaultAuditConnector) {
```

Comment:
Question: Do we need Inject even there are no paramenters?

Reply:
No, I think it will be useless in that case.
But here I am injecting `auditConnector`, so I need it